change to Hub generated open Event (#574)
HL7/fhircast-docs: input/pagecontent/2-4-Subscribing.md

Upon receiving subscription or unsubscription requests, the Hub SHALL respond to a subscription request with an appropriate HTTP response.

> If a Hub refuses the request or finds any errors in the subscription request, an appropriate HTTP error response code (4xx or 5xx) SHALL be returned. In the event of an error, the Hub SHOULD return a description of the error in the response body as plain text, to be used by the client developer to understand the error. This is not meant to be shown to the end user. Hubs MAY decide to reject some subscription requests based on their own policies. For example, a Hub may require that all applications subscribe to the same set of events in lieu of [deriving open events](https://build.fhir.org/ig/HL7/fhircast-docs/2-5-EventNotification.html#hub-generated-open-events).

Reviewer: ```suggestion
If a Hub refuses the request or finds any errors in the subscription request, an appropriate HTTP error response code (4xx or 5xx) SHALL be returned. In the event of an error, the Hub SHOULD return a description of the error in the response body as plain text, to be used by the client developer to understand the error. This is not meant to be shown to the end user. Hubs MAY decide to reject some subscription requests based on their own policies. For example, a Hub may require that all applications subscribe to the same set of events in lieu of [deriving open events](2-5-EventNotification.html#hub-generated-open-events).

```

Don't use absolute urls, otherwise, when we publish the spec, we'll have links to the build version of the spec.

> If a Hub refuses the request or finds any errors in the subscription request, an appropriate HTTP error response code (4xx or 5xx) SHALL be returned. In the event of an error, the Hub SHOULD return a description of the error in the response body as plain text, to be used by the client developer to understand the error. This is not meant to be shown to the end user. Hubs MAY decide to reject some subscription requests based on their own policies, for example, in lieu of a Hub generated open events it may require that all applications subscribe to the same set of events. This can be found in [Hub Generated open Events](https://build.fhir.org/ig/HL7/fhircast-docs/2-5-EventNotification.html#hub-generated-open-events).

Reviewer: ```suggestion
If a Hub refuses the request or finds any errors in the subscription request, an appropriate HTTP error response code (4xx or 5xx) SHALL be returned. In the event of an error, the Hub SHOULD return a description of the error in the response body as plain text, to be used by the client developer to understand the error. This is not meant to be shown to the end user. Hubs MAY decide to reject some subscription requests based on their own policies. For example, a Hub may require that all applications subscribe to the same set of events in lieu of [deriving open events](https://build.fhir.org/ig/HL7/fhircast-docs/2-5-EventNotification.html#hub-generated-open-events).
```
